notifications/push/fcm/notification.go

```diff
@@ -25,30 +26,35 @@ func NewNotification(key string) NotificationConstructor {
 		client := fcm.NewFcmClient(key).
 			SetDelayWhileIdle(true).
 			SetContentAvailable(true).
+			SetPriority(fcm.Priority_HIGH).
```

Comment:
makes sense to mention it in PR description, what kind of effect it will have?

Reply:
Done, I'll also add a comment in the code. It allows Firebase to call the new background task in the Android app which will take care of displaying the PN.